### ISSUE TITLE
Error sending traces

### ISSUE THREAD

**Issue by carlosalberto** [lightstep/opentelemetry-exporter-java#42]
It was reported we got this error:

```sh
WARN 14997 --- [main] c.l.o.exporter.LightstepSpanExporter : Failed to post spans to collector. 
Response{protocol=h2, code=400, message=, url=https://collector-grpc.lightstep.com/api/v2/reports}
```

This seems to be specially happening when configuring values through env vars and something that didn't happen using v0.2.0, so it's a regression caused by the last configuration refactoring, I suspect.

Assigning this to @malafeev for initial research ;)

**Comment by malafeev** [comment]
export OTA_EXPORTER_JAR="/path/to/lightstep-opentelemetry-auto-exporter-0.2.2.jar"
export LIGHTSTEP_COMPONENT_NAME="my-app-name"
export LIGHTSTEP_ACCESS_TOKEN="my-access-token"
java -javaagent:/path/to/opentelemetry-auto-0.2.2.jar \
-jar my_application.jar

Following the installation guide with pet clinic: https://docs.lightstep.com/otel/java-installation-guide

**Comment by malafeev** [comment]
Issue is fixed via #43  but
I'm going to update README with examples as part of this.

**Comment by malafeev** [comment]
done
